assertions/src/main/java/io/camunda/zeebe/process/test/inspections/InspectionUtility.java

```diff
@@ -33,4 +33,8 @@ public static ProcessInstanceInspections findProcessInstances() {
     return new ProcessInstanceInspections(
         StreamFilter.processInstance(BpmnAssert.getRecordStream()));
   }
+
+  public static DeploymentInspections findProcessDefinitions() {
+    return new DeploymentInspections(StreamFilter.deploymentRecords(BpmnAssert.getRecordStream()));
+  }
```

Comment:
❌ Currently you're using the Deployment. I learned (after our chat yesterday) that we are removing the resources from the `Deployment.CREATED` event, in order to allow for larger deployments.

Instead, it's more clean to get the resource from the `Process.CREATED` event. Obviously this also requires some renaming of classes and splitting the DMN and BPMN parts. Personally I'd propose to leave DMN out of this PR as DMN isn't really supported in ZPT currently 😅

Reply:
Fixed in https://github.com/camunda/zeebe-process-test/pull/813/commits/d139f72b73a5eabc4519b65967b7cec4e3d52d8f